doc/man3/SSL_get_handshake_rtt.pod

```diff
@@ -0,0 +1,60 @@
+=pod
+
+=head1 NAME
+
+SSL_get_handshake_rtt
+- get round trip time for SSL Handshake
+
+=head1 SYNOPSIS
+
+ #include <openssl/ssl.h>
+
+ int SSL_get_handshake_rtt(const SSL *s, uint64_t *rtt);
+
+=head1 DESCRIPTION
+
+SSL_get_handshake_rtt() retrieves the round-trip time (RTT) for I<ssl>.
+
+This metric is represented in microseconds (us) as a uint64_t data type.
+
+=head1 NOTES
+
+This metric is created by taking two timestamps during the handshake and providing the
+difference between these two times.
+
+When acting as the server, one timestamp is taken when the server is finished
+writing to the client. This is during the ServerFinished in TLS 1.3 and 
+ServerHelloDone in TLS 1.2. The other timestamp is taken when the server is done reading 
+the client's response. This is after the client has responded with ClientFinished.
+
+When acting as the client, one timestamp is taken when the client is finished writing 
+the ClientHello and early data (if any). The other is taken when client is done reading the server's response.
+This is after ServerFinished in TLS 1.3 and after ServerHelloDone in TLS 1.2.
+
+In addition to network propagation delay and network stack overhead, this metric 
+includes processing time on both endpoints, as this is based on TLS protocol-level 
+messages and the TLS protocol is not designed to measure network timings. In some cases the 
+processing time can be significant, especially when the processing includes asymmetric cryptographic 
+operations.
+
+=head1 RETURN VALUES
+
+Returns 1 if the TLS handshake round-trip time is successfully retrieved.
+Returns 0 if the TLS handshake round-trip time is not yet determined or if an error occurred in 
```

Comment:
Please distinguish the cases
* "not yet determined" (return code 0)
* "an error occurred" (return code -1)

Reply:
I am not sure this is that much useful distinction. So no, I would not require this.

Reply:
From the user's perspective it is relevant to know whether an error is recoverable (i.e., whether it is worth retrying later) or not.
And it is easy to achieve -  simply change the check in `SSL_get_handshake_rtt()`
```
    if (sc == NULL || sc->ts_msg_write.t <= 0 || sc->ts_msg_read.t <= 0
            || sc->ts_msg_read.t < sc->ts_msg_write.t)
        return 0;
```
to
```
    if (sc == NULL)
        return -1;
    if (sc->ts_msg_write.t <= 0 || sc->ts_msg_read.t <= 0)
        return 0;
    if (sc->ts_msg_read.t < sc->ts_msg_write.t)
        return -1;
```

Reply:
Sure, I'll make that code change. 

And rewrite the return values to this
```
Returns 1 if the TLS handshake RTT is successfully retrieved.
Returns 0 if the TLS handshake RTT cannot yet determined.
Returns -1 if, while retrieving the TLS handshake RTT, an error occurs.
```